pkg/provider/provider.go

```diff
@@ -100,13 +102,15 @@ type kubeProvider struct {
 
 	clientSet  *clients.DynamicClientSet
 	k8sVersion cluster.ServerVersion
+	resources  k8sopenapi.Resources
 }
 
 var _ pulumirpc.ResourceProviderServer = (*kubeProvider)(nil)
 
 func makeKubeProvider(
 	host *provider.HostClient, name, version string,
 ) (pulumirpc.ResourceProviderServer, error) {
+	cmdutil.InitProfiling("k8s-provider")
```

Comment:
This and the other profiling-related change were accidental. I will remove them prior to merging.